### PR TITLE
Feature/tss 1093/review access control

### PR DESCRIPTION
**Pull request by ferozerub**

### CODE REVIEW

File: tests/feedback/test_views.py [hunk 1]
```diff
@@ -11,7 +16,14 @@
...
 
 
-class MyTestCase(APITestMixin, APITestCase):
+class FeedbackTestCase(APITestMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.user1 = create_test_user(sso_user_id=self.sso_user_data_1["user_id"])
+        self.user2 = create_test_user(sso_user_id=self.sso_user_data_2["user_id"])
+        self.api_client1 = self.create_api_client(user=self.user1)
+        self.api_client2 = self.create_api_client(user=self.user2)
```
Comment (chris-pettinga) [inline tests/feedback/test_views.py:25]: Looks like we don't use this further on down?

**Review by chris-pettinga — CHANGES_REQUESTED** [review]

**Review by chris-pettinga — APPROVED** [review]

File: tests/feedback/test_views.py (hunk 1 above)
Comment (ferozerub) [reply tests/feedback/test_views.py]: Agreed - removed the second api client

File: api/core/permissions.py [hunk 2]
```diff
@@ -8,7 +8,23 @@ class IsAuthenticated(BasePermission):
...
+class IsCreatorOrReadOnly(BasePermission):
+    """
+    Object-level permission to only allow creators of an object to edit it.
+    Assumes the model instance has an `created_by` attribute.
+    """
+
+    def has_object_permission(self, request, view, obj):
+        # Read permissions are allowed to any request,
+        # so we'll always allow GET, HEAD or OPTIONS requests.
+        if request.method in SAFE_METHODS:
```
Comment (osimuka) [inline api/core/permissions.py:26]: if this is a get request, in DRF which is also `retrieve` & `list` it will return True

**Review by Santino-Trade — APPROVED** [review]

File: api/core/permissions.py (hunk 2 above)
Comment (osimuka) [reply api/core/permissions.py:26]: is this what we want here?

**Review by osimuka — APPROVED** [review]

**Pull request merged**